components/form/FormItem.razor.cs

```diff
@@ -31,6 +31,9 @@ public partial class FormItem : AntDomComponentBase, IFormItem
         [CascadingParameter(Name = "FormItem")]
         private IFormItem ParentFormItem { get; set; }
 
+        [Parameter]
+        public string DataIndex { get; set; }
```

Comment:
It is better to change the attribute name to `name`, It will also be used later in our ssr form support. https://github.com/ant-design-blazor/ant-design-blazor/pull/3580